### ISSUE TITLE
Some "Action" links should have the role=button

### ISSUE THREAD

**Issue by EmmanuelDemey** [got5/KAWWA#109]
Links like 'add to list" should have the role=button. because the Jaws/NVDA tool should not list this link in its tool. This link will not reload the page. It is just an action element.
![image](https://f.cloud.github.com/assets/555768/1427820/34099e4c-4094-11e3-80f8-dafbc8253af7.png)
